README.md

```diff
@@ -68,24 +68,24 @@ in Sass):
 #### `asset-path($relative-asset-path)`
 Returns a string to the asset.
 
-* `asset-path("rails.png")` becomes `"/assets/rails.png"`
```

Comment:
The idea of this line is to tell that the call of `asset-path("rails.png")` will generate that string. So maybe we should change `becomes` with something else.